content/fast-api-schemathesis.md

```diff
@@ -0,0 +1,242 @@
+Reproduce with:
+    curl -X POST -H 'Content-Type: application/json' -d '{"author": "", "category": "", "topic": ""}' http://127.0.0.1:8000/tutorials/
+================================================================================================================== SUMMARY ==========
+Performed checks:
+    not_a_server_error                              5 / 5 passed          PASSED
+    status_code_conformance                         0 / 5 passed          FAILED
+    content_type_conformance                        5 / 5 passed          PASSED
+    response_headers_conformance                    5 / 5 passed          PASSED
+    response_schema_conformance                     5 / 5 passed          PASSED
+    negative_data_rejection                         5 / 5 passed          PASSED
+    ignored_auth                                    5 / 5 passed          PASSED
+```
+
+More tests are now passing but the report reflects that even if you handled the db integrity error, you haven't documented it.
+You can add a response code to your route like this:
+```python
+class Error(BaseModel):
+    detail: str
+
+# Response code is included in the decorator
+@router.post("/tutorials/", responses={409: {"model": Error}})
+def create_tutorial(tutorial_data: TutorialCreate, db: Session = Depends(get_db)):
+    try:
+        tutorial = Tutorial(**tutorial_data.dict())
+        db.add(tutorial)
+        db.commit()
+    except exc.IntegrityError:
+        raise HTTPException(
+            status_code=409,
+            detail="A tutorial with this topic already exists",
+        )
+```
+
+After a re-run, all test cases should pass. By default, `schemathesis` generates 100 test cases, but you can adjust that.
+
+```shell
+POST /tutorials/ .                                                                                                                                                                                    
+Performed checks:
+    not_a_server_error                              100 / 100 passed          PASSED
+    status_code_conformance                         100 / 100 passed          PASSED
+    content_type_conformance                        100 / 100 passed          PASSED
+    response_headers_conformance                    100 / 100 passed          PASSED
+    response_schema_conformance                     100 / 100 passed          PASSED
+    negative_data_rejection                         100 / 100 passed          PASSED
+    ignored_auth                                    100 / 100 passed          PASSED
+```
+
+That's it! On a very simple example, you have already improved your data validation and documented new error codes.
+
+### Limitations
+
+Just from the first few tries of the library, I noticed a few limitations:
+
+- supporting `openapi 3.1` is still an experimental feature, which you have to enable
+- when running the tests via `pytest` the summary of the performed checks cannot be enabled. You can set the verbosity of 
```

Comment:
That would be a nice feature to add to Schemathesis! :)

P.S. Great post :) Would you mind mentioning your post in Schemathesis docs?